docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md

```diff
@@ -97,6 +103,8 @@ Here's how it works:
 
 With this deployment mode, the cluster automatically keeps specific rows and all the table shards (known as tablets) in the specified region. In addition to complying with data sovereignty requirements, you also get low-latency access to data from users in the region while maintaining transactional consistency semantics.
 
+In YugabyteDB Managed, a Partition by region cluster consists of a primary region where all tables that aren't geo-partitioned (that is, don't reside in a tablespace) reside, and any number of additional regions where you can store partitioned data, whether it's to reduce latencies or comply with data sovereignty requirements.
```

Comment:
It can have more than one primary region. This is a UI limitation, but technically we can have the data spread across regions by default as well if the user so desires

Reply:
Added 'initially'